<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSelfflyWidget</class>
 <widget class="QWidget" name="QSelfflyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>135</height>
    </rect>
   </property>
   <property name="title">
    <string>纵向设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>内环灵敏度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QSlider" name="horizontalSlider_THETA2ELE">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_THETA2ELE">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>外环灵敏度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QSlider" name="horizontalSlider_VX2ELE">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_VX2ELE">
      <property name="text">
       <string>0  </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>速度范围</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_ELEMIN"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_232">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_ELEMAX"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_233">
      <property name="text">
       <string>度/秒</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_AUTO_ELEASK">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_AUTO_ELESET">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_9</zorder>
   <zorder>label_8</zorder>
   <zorder>horizontalSlider_THETA2ELE</zorder>
   <zorder>horizontalSlider_VX2ELE</zorder>
   <zorder>label_THETA2ELE</zorder>
   <zorder>label_VX2ELE</zorder>
   <zorder>label_13</zorder>
   <zorder>label_233</zorder>
   <zorder>label_232</zorder>
   <zorder>lineEdit_ELEMAX</zorder>
   <zorder>lineEdit_ELEMIN</zorder>
   <zorder>pushButton_AUTO_ELEASK</zorder>
   <zorder>pushButton_AUTO_ELESET</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_73">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>航向设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_12">
    <item row="1" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>速度范围</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>航向灵敏度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_236">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_RUDMIN"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_AUTO_RUDASK">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_AUTO_RUD">
      <property name="text">
       <string>0  </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_AUTO_RUDSET">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_237">
      <property name="text">
       <string>度/秒</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_RUDMAX"/>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QSlider" name="horizontalSlider_AUTO_RUD">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_74">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>升降设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_13">
    <item row="0" column="5">
     <widget class="QLabel" name="label_AUTO_COL">
      <property name="text">
       <string>0  </string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_COLMAX"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_COLMIN"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_AUTO_COLASK">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_AUTO_COLSET">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_239">
      <property name="text">
       <string>度/秒</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_238">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>升降灵敏度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>速度范围</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QSlider" name="horizontalSlider_AUTO_COL">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_71">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>331</width>
     <height>135</height>
    </rect>
   </property>
   <property name="title">
    <string>横向设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>内环灵敏度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QSlider" name="horizontalSlider_PHI2AIL">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_PHI2AIL">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>外环灵敏度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QSlider" name="horizontalSlider_VY2AIL">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_VY2AIL">
      <property name="text">
       <string>0  </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>速度范围</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_AILMIN"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_234">
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_AILMAX"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_235">
      <property name="text">
       <string>度/秒</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_AUTO_AILASK">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_AUTO_AILSET">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
